availability.py

```diff
@@ -101,7 +101,10 @@ def get_location(self):
         if self.event is not None:
             location = self.event.decoded('location')
             if location is not None:
-                return location.decode("utf-8", "ignore")
+                locationStr=location.decode("utf-8", "ignore")
+                locationStr.replace("_"," ")
+                locationStr.replace("Konf ","Konferensrum ")
+                return locationStr
```

Comment:
This code does not work, replace returns a new string the original string is unmodified. 

The lines could be simplified with:
return locationStr.replace("_"," ").replace("Konf ","Konferensrum ")